cfnstack/provisioner.go

```diff
@@ -231,3 +231,67 @@ func (c *Destroyer) Destroy() error {
 	_, err := cfSvc.DeleteStack(dreq)
 	return err
 }
+
+func (c *Provisioner) StreamCloudFormationNested(q chan struct{}, f *cloudformation.CloudFormation, stackId string, headStackName string, t time.Time) error {
```

Comment:
Nit: I'd like its name to be `StreamEventsNested` after https://github.com/kubernetes-incubator/kube-aws/pull/801/files#r129498094